app/models/persistent_volume_claim.rb

```diff
@@ -0,0 +1,6 @@
+class PersistentVolumeClaim < ActiveRecord::Base
```

Comment:
Agreed in general.  Having this scoped to containers in some way would be better.  Naming is always hard, though.

We have to work out the details around having models that pertain to only a particular provider type living only under that provider in `app/models`.  We might have examples of that already, not sure.  If so, then the best solution might be to just move all these under `ManageIQ::Providers::ContainerManager` (not in this PR, but eventually).